src/Core/Form/IdentifiableObject/CommandBuilder/Product/BasicInformationCommandsBuilder.php

```diff
@@ -41,21 +41,21 @@ class BasicInformationCommandsBuilder implements ProductCommandsBuilderInterface
      */
     public function buildCommands(ProductId $productId, array $formData): array
     {
-        if (!isset($formData['basic']) && !isset($formData['header']['name'])) {
+        if (empty($formData['description']) && empty($formData['header']['name'])) {
             return [];
         }
 
-        $basicData = $formData['basic'] ?? [];
+        $descriptionData = $formData['description'] ?? [];
         $command = new UpdateProductBasicInformationCommand($productId->getValue());
 
         if (isset($formData['header']['name'])) {
             $command->setLocalizedNames($formData['header']['name']);
         }
-        if (isset($basicData['description'])) {
-            $command->setLocalizedDescriptions($basicData['description']);
+        if (isset($descriptionData['description'])) {
```

Comment:
Yes, I would also prefer to use `informations` to avoid having it twice, but the main reason here is that the name of the first form types, match the names of the tabs I think it's the easiest way to keep a logical meaning and to find the appropriate path more easily

In the new page structure the former tab `Basic settings` has been split into two tabs `Description` and `Specifications` (see screenshot)

<img width="1001" alt="Capture d’écran 2021-07-26 à 11 01 11" src="https://user-images.githubusercontent.com/13801017/126963094-ccee01c2-ca46-4d21-887e-8a52dd14ad50.png">

So keeping this logic seems more important than avoiding duplicity. Regarding the CQRS commands they are less coupled with presentation than the form so that's why I didn't rename them for now, but that's a possibility I don't completely ignore. Maybe some commands will be renamed to keep the consistency, we didn't decide yet.

So TLDR; if the tab was named `Informations` I wouldn't mind using the same wording in the form type, but that's a decision from @PrestaShop/product-team @MatShir or @TristanLDD